_includes/tutorials/ksql-heterogeneous-json/ksql/code/src/statements.sql

```diff
@@ -0,0 +1,18 @@
+CREATE STREAM DATA_STREAM ( 
+  JSONTypeOne VARCHAR, 
```

Comment:
This is just for consideration, would it be a valid name and would it be more helpful to change the word to a number, e.g. instead of `JSONTypeOne`, write `JSONType1`, etc....  The reasoning is it may be easier for a reader to more quickly digest, at least for me it is.

Reply:
ack